statServer/models/questions_model.ts

```diff
@@ -0,0 +1,88 @@
+/*
+El nuevo entrypoint tiene la siguiente estructura:
+
+GET /questions?since=2022-03-04
+
+{
+   "questionId": ".. ",  // Identificación de la pregunta 
+   "question": ".. ",  // Texto de la pregunta en español
+   "attempts":  #  // Número de veces que la pregunta se ha intentado 
+   "avgLatency": # // Tiempo promedio en segundo que la gente necesita para responder esta pregunta
+   "successProb": # //  La probabilidad de tener éxito respondiendo esta pregunta (valor entre cero y uno)
+}
+since: fecha para considerar los datos sólo después de la fecha dada, es opcional, si no está se consideran todos los datos disponibles.
+
+Para los textos de las preguntas se va a usar el resultado del issue Issue 54
+*/
+
+import { Questions } from '../types/questions';
+import { db } from "../db";
+import { RowDataPacket } from "mysql2";
+
+
+export const getQuetionsByDates = (callback: Function, dates: any) => {
+
+    let { since, to } = dates;
+    // console.log(since)
```

Comment:
No veo ninguna razón para recibir el parametro dates en vez de directamente since y to. Por favor recibir los dos parametros

Reply:
Se estructura para que quede cada uno como parametro